crates/bevy_ecs/src/event/iterators.rs

```diff
@@ -233,6 +235,10 @@ impl<'a, E: Event> EventParIter<'a, E> {
                     });
                 }
             });
+
+            // If we get to hear we're guaranteed to have seen all events
```

Comment:
```suggestion
            // Events are guaranteed to be read at this point.
```
Just a small wording suggestion. Although I feel like this explanation could be already implied from the code, right?